pinot-core/src/test/java/org/apache/pinot/core/data/function/InbuiltFunctionEvaluatorTest.java

```diff
@@ -120,10 +128,27 @@ public void testStateSharedBetweenRowsForExecution()
     assertEquals(evaluator.evaluate(row), "test test test ");
   }
 
+  @Test
+  public void testExceptionDuringInbuiltFunctionEvaluator()
+      throws Exception {
+    String expression = "fromDateTime('2020-01-01T00:00:00Z', \"invalid_identifier\")";
+    InbuiltFunctionEvaluator evaluator = new InbuiltFunctionEvaluator(expression);
+    assertEquals(1, evaluator.getArguments().size());
```

Comment:
First argument is actual, second argument is expected. It will be easier for debugging when it fails
```suggestion
    assertEquals(evaluator.getArguments().size(), 1);
```

Reply:
geeze.. this is revert convention from junit!!!!
http://junit.sourceforge.net/javadoc/org/junit/Assert.html
https://www.javadoc.io/doc/org.testng/testng/6.8.17/org/testng/Assert.html